packages/react/src/components/SuiteHeader/util/IdleTimer.js

```diff
@@ -65,7 +66,7 @@ class IdleTimer {
       document.cookie.split('; ').reduce((r, v) => {
         const parts = v.split('=');
         return parts[0] === this.COOKIE_NAME ? decodeURIComponent(parts[1]) : r;
-      }, Date.now()),
+      }, NaN),
```

Comment:
Thanks, I replaced the existing code with your version of the `getUserInactivityTimeout` method. However, I needed to add the `?` optional chaining to `cookie.split` to avoid throwing an error when the cookie has been deleted.
So, the before the `return` statement, the line is now like this:
```
const cookieValue = cookie?.split('=')[1];
```